README.md

```diff
@@ -1,7 +1,107 @@
-# th2 codec hand (0.1.3)
+# th2 codec hand (0.1.4)
+
+This microservice can decode messages received from component th2-hand.
+Example of th2-hand output:
+
+```json
+{
+  "ScriptOutputCode": "SUCCESS",
+  "ErrorText": "",
+  "ActionResults": [
+    {
+      "id": "th2_hand_action_id1",
+      "data": "8=FIXT.1.1\u00019=259\u000135=D\u0001..."
+    }
+  ],
+  "RhSessionId": "th2_hand"
+}
+```
+th2-hand codec will decode this kind of messages to display separated fields
+in the report-viewer and some fields will be passed as raw and can be accessible 
+other codecs in codec-pipeline.
+
+## Configuration
+
+Main configuration is done via setting following properties in custom configuration block:
```

Comment:
"Main configuration is done by setting the following properties in custom configuration block:"